<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Profile:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="6">
    <widget class="QLineEdit" name="phoneLineEdit"/>
   </item>
   <item row="1" column="2" colspan="6">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="PClassLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="4">
    <widget class="QDateEdit" name="birthDateEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="8">
    <widget class="QLineEdit" name="commentLineEdit">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string comment="Additional comments" extracomment="Additional comments"/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Additional comments</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Contacts:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date of birth</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sexLabel">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="6">
    <widget class="QLineEdit" name="emailLineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="genderComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="7">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="6">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Student</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Teacher</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Academic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Study office</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Service staff</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lecturer</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cacncel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
